src/governance.cpp

```diff
@@ -933,26 +933,32 @@ bool CGovernanceManager::ProcessVote(CNode* pfrom, const CGovernanceVote& vote,
 {
     ENTER_CRITICAL_SECTION(cs);
     uint256 nHashVote = vote.GetHash();
-    if(mapInvalidVotes.HasKey(nHashVote)) {
+    uint256 nHashGovobj = vote.GetParentHash();
+
+    if(cmapVoteToObject.HasKey(nHashVote)) {
+        LogPrint("gobject", "CGovernanceObject::ProcessVote -- ignoring existing vote %s for object %s\n", nHashVote.ToString(), nHashGovobj.ToString());
```

Comment:
Is this related to the refactoring or is this an additional fix for something?

Reply:
Ooops, this should definitely be in its own separate commit. Will fix.